src/emulator/instruction.ts

```diff
@@ -68,11 +70,14 @@ export const enum AddressingMode {
 
     imm8,
     imm8io,
+    imm8sign,
```

Comment:
Für den sehe ich nicht so wirklich die Existenzberechtigung, es sei denn wir dekodieren das im Disassembler auch mit Vorzeichen 😛

Reply:
den hab ich eingeführt damit in der CPU über den Addressierungsmodus das extendSign8() aufrufe. Damit kann ich das für ADD,  LDS und JR nutzen, die auch einen vorzeichenbehaften Wert übergeben bekommen können.

Reply:
Bei der Addition brauchst Du kein `extendSign`, da Du da zwei Werte gleicher Breite addierst, da stimmt das Zweierkomplement schon. Im Code finde ich das auch nicht.

Bei den anderen beiden könnte das `extendSign` so bisher auch einfach im Code stehen. Der einzige Mehrwert, den ich sehe ist, dass man mit der Information das Zweierkomplement im Disassembler dekodieren könnte.

Reply:
OpCode 0xe8 ADD SP,r8 ist das mit dem signed int beim Add. ;)

Reply:
Hmja, dann macht das im Adressierungsmodus Sinn, das war mir durchgerutscht. Dann fände ich es nur nett, wenn wir die Information beim Disassemblieren benutzen und das Zweierkomplement in eine Zahl mit Vorzeichen konvertieren würden, wenn wir die Information schon haben.